Tests/GDSAnalyticsTests/Screens/ScreenViewTests.swift

```diff
@@ -0,0 +1,35 @@
+import GDSAnalytics
+import XCTest
+
+final class ScreenViewTests: XCTestCase {
+    func testInitialisation() {
+        let view = ScreenView(screen: MockScreen.welcome,
+                              titleKey: "Welcome to this app")
+        
+        XCTAssertEqual(view.screen, MockScreen.welcome)
+        XCTAssertEqual(view.title, "welcome to this app")
```

Comment:
I would've thought that this would fail because of the lowercase 'W' .. how comes it doesn't pick this up?

Reply:
The test passed because of this line
`self.title = titleKey.englishString(variableKeys, bundle: bundle).formattedAsParameter`
transform the string to lowercase :)